src/tests/testHttpRequest.cc

```diff
@@ -46,7 +46,7 @@ testHttpRequest::testCreateFromUrl()
     /* vanilla url, implict method */
     unsigned short expected_port;
     SBuf url("http://foo:90/bar");
-    const MasterXaction::Pointer mx = new MasterXaction(XactionInitiator::initClient);
+    const MasterXaction::Pointer mx = new MasterXaction(nullptr);
```

Comment:
This and two similar changes seem to contradict the new MasterXaction construstor descriptions. These test transactions are not associated with a listening port and, hence, should not be constructed using a port pointer, and may continue to use initClient instead, right?